### PR TITLE
[ENHANCEMENT] Variable UX fixes

### PR DESCRIPTION
**Pull request by saminzadeh**
- Changing some variable copy in the editor (e.g. drop the "Variable" prefix)
image
- Better names of the Variable plugins
image
- Tooltip to tell user to press enter in the Static List (otherwise the values will get erased)
image
- Ability to copy the variables values:
image
- Allow pasting comma separated values into the Static List options Input 
- Remove the "Hide variables" option in Edit mode
image
- [BugFix] Custom Label is now used for TextVariables

Signed-off-by: Shan Aminzadeh <shan.aminzadeh@chronosphere.io>

### CODE REVIEW

File: ui/dashboards/src/components/Variables/VariableEditor.tsx [hunk 1]
```diff
@@ -143,7 +144,7 @@ export function VariableEditor(props: {
               borderBottom: (theme) => `1px solid ${theme.palette.divider}`,
             }}
           >
-            <Typography variant="h2">Template Variables</Typography>
+            <Typography variant="h2">Variables</Typography>
```
Comment (cndonovan) [inline ui/dashboards/src/components/Variables/VariableEditor.tsx:147]: 👏 I like this change

**Review by cndonovan — APPROVED** [review]

File: ui/prometheus-plugin/plugin.json [hunk 2]
```diff
@@ -9,31 +9,31 @@
...
+          "name": "[Static] Custom List",
           "description": "Static list variables allow you to define an array of values"
         }
       },
       {
         "pluginType": "Variable",
         "kind": "PrometheusLabelValuesVariable",
         "display": {
-          "name": "PrometheusLabelValuesVariable",
+          "name": "[Prometheus] Label Values",
```
Comment (sjcobb) [inline ui/prometheus-plugin/plugin.json:20]: Love these new names, much more readable

**Review by sjcobb — APPROVED** [review]

File: ui/dashboards/src/components/Variables/VariableEditorForm/VariableEditorForm.tsx [hunk 3]
```diff
@@ -80,6 +86,28 @@ function VariableListPreview({ definition }: { definition: ListVariableDefinitio
...
+                await navigator.clipboard.writeText(data.map((d) => d.label).join(','));
+                alert('Copied to clipboard!');
+              }
+            }}
+            size="small"
+          >
+            <Clipboard />
+          </IconButton>
+        </Tooltip>
+        <IconButton size="small"></IconButton>
```
[the remark this answers is not in the document]
Comment (saminzadeh) [reply ui/dashboards/src/components/Variables/VariableEditorForm/VariableEditorForm.tsx:109]: Nice catch!

File: ui/dashboards/src/components/Variables/VariableList.tsx [hunk 4]
```diff
@@ -32,8 +31,7 @@ export function TemplateVariableList(props: TemplateVariableListProps) {
   const [isPin, setIsPin] = useState(props.initialVariableIsSticky);
   const variableDefinitions = useTemplateVariableDefinitions();
   const { isEditMode } = useEditMode();
-  const [showVariablesInEditMode, setShowVariablesInEditMode] = useState(true);
-  const showVariables = isEditMode ? showVariablesInEditMode : true;
+  const showVariables = isEditMode;
```
Comment (eunicorn) [inline ui/dashboards/src/components/Variables/VariableList.tsx:34]: shouldn't we always show variables even in view mode?

**Comment by eunicorn** [comment]
What do you think about calling just "List" or "Text" for type in the table as well? 
image

**Review by eunicorn — APPROVED** [review]

File: ui/prometheus-plugin/src/plugins/variable.tsx [hunk 5]
```diff
@@ -40,13 +40,26 @@ function StaticListVariableOptionEditor(props: OptionsEditorProps<StaticListVari
   return (
     <div>
       <Autocomplete
+        onPaste={(e) => {
+          // Append new values on paste
+          const v = e.clipboardData.getData('text/plain');
+          if (v) {
+            const values = v.split(',');
+            onChange(null, value.concat(values));
```
Comment (LukeTillman) [inline ui/prometheus-plugin/src/plugins/variable.tsx:48]: Maybe a "future" thing, but should we be trying to de-dupe values or anything in this case? I guess there probably isn't anything stopping someone from entering duplicates manually now, so maybe a future validation topic.

**Review by LukeTillman — APPROVED** [review]

**Pull request merged**